apps/labs/tailwind.config.js

```diff
@@ -3,7 +3,7 @@ const { join } = require('path');
 
 module.exports = {
   content: [
-    join(__dirname, '**/!(*.stories|*.spec).{ts,tsx,html}'),
+    join(__dirname, '**/!(*.stories|*.spec).{ts,tsx,html,md,mdx}'),
```

Comment:
This change adds all `.md` and `.mdx` files to Tailwind CSS processing. I think all that means is that Tailwind will scan all of those files for class names so that it produces a CSS file that's as minimal as possible. But I'm not 100% sure, so we'll need to be sure to take an extra hard look at the Labs site.

There were alternatives to doing this. One possibility was to keep the `<style>`-tag styles that Anirudh had added to his blog post. But this made me a little nervous because I'm not entirely sure if Next.js removes them afterwards. Furthermore, Anirudh's styles didn't look quite as good in the new website because the text width was different, which caused the "cons" headings to be misaligned, depending on screen width. I also suspect that being able to use Tailwind styles in blog post MD/X files might be useful in the future, especially if we have more blog posts with special, one-off styling. 

@bskinn, I did not make this change Consulting's Tailwind configuration file, but perhaps we should for consistency? That is... once we establish that the change doesn't mess things up.

Reply:
@bskinn Pinging just to surface this comment for you. :)